README.md

```diff
@@ -15,11 +15,11 @@ gem 'graphlient'
 
 ## Usage
 
-Graphlient always uses named queries, which means you need to start your query definition with using `query` which gets a symbol for query name as argument and a block for actual query definition. There are 3 different ways to use this library.
+You need to start your query definition with using `query` which gets a block for actual query definition. There are 3 different ways to use this library.
```

Comment:
Nitpicking: for a doc it's better to use fewer words and not being so directed. Eg. "Start a query definition ..."